tests/test_service.rb

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/service'
+require 'pp'
+
+include Cisco
+
+# TestService - Minitest for feature Service node Utility class
+class TestService < CiscoTestCase
+  @skip_unless_supported = 'service'
+  ###################
+  # Service tests   #
+  ###################
+
+  def test_clear_status
+    Service.clear_status
+    assert_equal(Service.upgraded?, false)
+  end
+
+  def test_delete
```

Comment:
Couple thoughts on `test_delete` and `test_delete_boot`.

- Let's leave them commented out for now since these are pretty destructive and they will get run automatically in our CI pipeline once this code is merged into `develop`.
- We should add tests to verify that we get a `Cisco::CliError` for `self.delete` when the images or media indexes to a non-existent image.
  - Example minitest: https://github.com/cisco/cisco-network-node-utils/blob/45aaac5fb9098b7a9d8322268e390b25ed324c05/tests/test_encapsulation.rb#L70